lib/collector.js

```diff
@@ -7,17 +7,35 @@ class Collector {
   count = 0;
   exact = true;
   timeout = 0;
+  isRejectable = true;
   #fulfill = null;
   #reject = null;
   #timer = null;
   #cause = null;
 
-  constructor(keys, { exact = true, timeout = 0 } = {}) {
+  constructor(keys, { exact = true, timeout = 0, isRejectable = true } = {}) {
     this.keys = keys;
+    this.isRejectable = isRejectable;
```

Comment:
We should check if `isRejectable` is boolean

Reply:
Fixed